Algorithm.CSharp/BacktestingBrokerageRegressionAlgorithm.cs

```diff
@@ -96,7 +96,7 @@ public override void OnData(Slice data)
                         from optionContract in chain.OrderByDescending(x => x.Strike)
                         where optionContract.Right == OptionRight.Call
                         where optionContract.Expiry == Time.Date
-                        where optionContract.Strike < chain.Underlying.Price
+                        where optionContract.ScaledStrike < chain.Underlying.Price
```

Comment:
Since this issue only really applies for 0.01%, let's revert the changes of the example algorithms to use `ScaledStrike` instead of `Strike`, except for the new algorithm reproducing the issue 👍